Branch-SDK/Branch-SDK/Core/Branch.m

```diff
@@ -354,6 +357,16 @@ - (BOOL)handleDeepLink:(NSURL *)url {
     return handled;
 }
 
+- (BOOL)continueUserActivity:(NSUserActivity *)userActivity {
+    NSString *linkIdentifier = [self.contentDiscoveryManager spotlightLinkIdentifierFromActivity:userActivity];
```

Comment:
Yes, this is pending the backend change. I thought I had put a TODO in here specifically stating as much, but apparently I forgot.